docs/learning/ui/HostAppUI.md

```diff
@@ -1,20 +1,20 @@
-# Creating an IModelApp that supports Extensible UI
+# Creating an iTwinApp that supports Extensible UI
```

Comment:
As confusing as it is we may want to leave that as IModelApp since that's still the class's name.